content/ADR-175-remove-assets-from-build-local-addressables.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: adr
+adr: 175
+title: Remove assets from Renderer's build. Addressables Integration
+date: 2023-01-16
+status: Review
+type: Standards Track
+spdx-license: CC0-1.0
+authors:
+  - ajimenezDCL
+  - mikhail-dcl
+---
+
+## Abstract
+This document describes an approach on how to make the reference's client lightweight by streaming the currently embedded assets. 
+
+## Problem
+Every nice UI, sound effect, particle or material we add to the renderer increases its size. This directly impact loading times or caching since the client's size will go above the max allowed for it (applied to WebGL, check [the PoC](https://github.com/2fd/poc-browser-cache) for details). The idea is to pack textures, audio files and other assets, and stream them in runtime. 
+
+## Solution
+### Pack the Assets
+Similar to what we are doing with scene assets, we can pack everything into one or multiple (to always be under the cache size limit) asset packs to be consumed by the client. This solution should be part of a different system than the one used for scene asset. Reasons why we can't reuse the system include, but are not limited to:
+   - The existing system is legacy both from the runtime and distribution sides (it is based on "Asset Bundles"). Some parts require special attention such as refactoring, and some do not, but we are not going to base the new system on the legacy foundation, neither we are going to address the old issues immediately
```

Comment:
```suggestion
   - The existing system is legacy both from the runtime and distribution sides (it is based on "Asset Bundles"). Some parts require special attention, such as refactoring, and some do not, but we are not going to ground the new system on the legacy foundation, neither to address the old issues immediately
```

Reply:
Do you think "ground" is a proper word? I think it has a slightly different connotation, and here we need something closer to "build" or "found"

Reply:
You're right. Lets use something closer to "build" or "found".